src/UserConsentListBuilder.php

```diff
@@ -65,10 +66,12 @@ public function buildHeader() {
    * {@inheritdoc}
    */
   public function buildRow(EntityInterface $entity) {
+    /** @var \Drupal\user\UserInterface $owner */
+    $owner = $entity->getOwner();
     /* @var $entity \Drupal\data_policy\Entity\UserConsentInterface */
     return [
       'id' => $entity->id(),
-      'user' => $entity->getOwner()->getDisplayName(),
+      'user' => ($owner instanceof UserInterface) ? $owner->getDisplayName() : '',
```

Comment:
When a user is deleted then would be better to show some text instead of an empty string e.g.:
```suggestion
      'user' => ($owner instanceof UserInterface) ? $owner->getDisplayName() : $this->t('Deleted user'),
```

Reply:
Changed